---
bool blue_tooth_ok
